Fix duplicate URI escape

RoiEXLab opened the pull request:
Some Time ago when I adopted 2FA for our forum I noticed that it was listed as "TripleA%20Forum" instead of "TripleA Forum".
the `replace(/\s/, '%20')` part replaces all spaces with `%20` which is then encoded by `encodeURIComponent` to `%2520`, so that is being removed

RoiEXLab commented on websockets.js:
```diff
@@ -24,7 +24,7 @@ Sockets.regenerate = function (socket, data, callback) {
 		const baseUrl = nconf.get('url').replace(/.*?:\/\//g, '');
-		const issuer = encodeURIComponent(meta.config.title.replace(/\s/, '%20')).replace('+', '%20');
+		const issuer = encodeURIComponent(meta.config.title).replace('+', '%20');
 		const account = encodeURIComponent(userslug + '@' + baseUrl).replace('+', '%20');
```
oh and btw. I don't think `encodeURIComponent` does ever encode spaces as `+` I think it always uses `%20`

julianlam replied:
As per 068081cc148fe2c1dd23d623cddd680b1016e5d7, the replacement of `+` is done for iOS compatibility, so it should be left alone :smile:

julianlam replied:
It defies logic, but only happens on iOS, so I will not tempt fate by removing it :grimacing:

RoiEXLab replied:
Probably a good call ^^

RoiEXLab commented:
@julianlam Interestingly enough this issue still persists after this change.
Could it be the case that `meta.config.title` already stores the title URI encoded?

julianlam commented:
Does this happen with certain devices? All devices?

The site title (with a space) is captured correctly on my device (FreeOTP on Android)

RoiEXLab commented:
I'm using Google Authenticator, but I checked the encoded URI directly and it still contains the %25 part, so the problem should occur regardless of the app.
Is there a way I could check the stored forum title in the database for comparison?

julianlam commented:
> I checked the encoded URI directly and it still contains the %25 part

What did you mean by this, that you looked at the text contained in the QR code?

RoiEXLab commented:
>  this, that you looked at the text c



> What did you mean by this, that you looked at the text contained in the QR code?

Yes.

Will check the db thingy later this day

julianlam commented:
> Yes.

I think that's to be expected, since a QR code is essentially a URI (at least, for as far as the OTP apps are concerned), so it should be escaped.

RoiEXLab commented:
A %20 is expected but not a %2520

julianlam commented:
Indeed! So you'll need to help me to identify where the %2520 (or %20) is coming from

RoiEXLab commented:
After connecting to the db `db.objects.find({ _key: 'config' }).pretty()` revealed that title is indeed `TripleA Forum`:
```json
"title" : "TripleA Forum",
```

However I realized that in the dashboard it says "Forum was last restarted 6 months ago", even though I know I restarted it a lot of times since then.
Looking at the forum logs I noticed the recent restart attempts were interrupted by a permission denied error because some folder belonged to root instead of the nodebb user.
So I fixed the permission problem which solved this issue and also fixed https://github.com/NodeBB/nodebb-plugin-write-api/issues/134#issuecomment-725884950
Nothing more to discuss here :)
